src/hooks/use-color-data.js

```diff
@@ -0,0 +1,34 @@
+import { useStaticQuery, graphql } from 'gatsby';
+
+const useColorData = () => {
+  const { allSassVarsJson } = useStaticQuery(
+    graphql`
+      query SassColors {
+        allSassVarsJson {
+          edges {
+            node {
+              description
+              context {
+                name
+                value
+              }
+              file {
+                name
+              }
+              group
+              type
+            }
+          }
+        }
+      }
+    `,
+  );
+
+  const sassColorData = allSassVarsJson.edges.filter((item) => {
```

Comment:
there's probably a way to modify the query to get data only from the right file

Reply:
https://graphql.org/learn/queries/#directives

So I looked into Directives and it can only do this via booleans, not by something like filename.

was there another idea you had about filtering out colors.scss data within the graphql?